knowledge-base/user-guides/view_only.md

```diff
@@ -15,7 +15,7 @@ Note : In order to create a viewonly wallet you need first to compile the last s
 
 - To create a view only wallet you will need to first create a "normal" wallet and get the associated viewkey and address. You can get them once logged in simplewallet by typing "*viewkey*" and "*address*". Note each of them carefully and exit simplewallet.
 
-- Launch a new instance of simplewallet by typing "*./simplewallet --generate-from-view-key yourAddress:yourViewKey:nameOfTheViewOnlyWallet*" where *youViewKey* is the view key you got from step 1 and *yourAddress* the associated address. The last part of the command is the name you want to give to you view only portfolio.
+- Launch a new instance of simplewallet by typing "`./simplewallet --generate-from-view-key yourAddress:yourViewKey:nameOfTheViewOnlyWallet`" where *yourViewKey* is the view key you got from step 1 and *yourAddress* the associated address. The last part of the command is the name you want to give to you view only portfolio.
```

Comment:
I'll merge this anyway, because "simplewallet" needs to change, so someone can change it after merge